jaegertracing-kubernetes-deployment-itest/src/main/java/io/jaegertracing/kubernetes/deployment/BaseETest.java

```diff
@@ -146,10 +174,17 @@ public void hitDependencyScreen() throws IOException {
     assertEquals(200, response.code());
   }
 
-  protected com.uber.jaeger.Tracer createTracer(String serviceName) {
+  protected com.uber.jaeger.Tracer createJaegerTracer(String serviceName) {
```

Comment:
Is there a reason to return `c.u.j.Tracer` instead of OpenTracing's ?

Reply:
IIRC later the tracer is closed to ensure all spans are reported

Reply:
This was there before - only different from OT Tracer is there is a close, which is being used, not sure if has an effect on the tests.

Reply:
By closing it we ensure that it reports immediately all it has.

Reply:
Ok thanks for the clarification.